packs/october.js

```diff
@@ -0,0 +1,39 @@
+  /** Additional description of a Lighthouse audit that tells the user how they can improve image loading by using webp in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'uses-webp-images': "Consider using a [plugin](https://octobercms.com/plugins?search=image) or service that will automatically convert your uploaded images to the optimal formats.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by lazy loading images that are initially offscreen in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'offscreen-images': "Consider installing a [image lazy-loading plugin](https://octobercms.com/plugins/) that provides the ability to defer any offscreen images, or switch to a theme that provides that functionality. Also consider using [the AMP plugin](https://octobercms.com/plugins?search=Accelerated+Mobile+Pages).",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve site loading performance by reducing the total bytes delivered by their page in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'total-byte-weight': "Consider showing excerpts in your post lists (e.g. via the more tag), reducing the number of posts shown on a given page, breaking your long posts into multiple pages, or using a plugin to lazy-load comments.",
+  /** Additional description of a Lighthouse audit that tells the user how they can improve performance by reducing the amount of render blocking resources present on their page, in the context of the October CMS CMS platform. This is displayed after a user expands the section to see more. No character length limits. Links in (parenthesis) become link texts to additional documentation. */
+  'render-blocking-resources': "A number of plugins that can help you [inline critical assets](https://octobercms.com/plugins?search=css). Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.",
```

Comment:
```suggestion
  'render-blocking-resources': "There are many plugins that help [inline critical assets](https://octobercms.com/plugins?search=css). Beware that optimizations provided by these plugins may break features of your theme or plugins, so you will likely need to make code changes.",
```

(avoid english idioms "a number of"; avoid "you")

Reply:
Removed idioms - may have missed some?